public/app/features/api-keys/state/actions.ts

```diff
@@ -29,28 +29,28 @@ const apiKeysLoaded = (apiKeys: ApiKey[]): LoadApiKeysAction => ({
 export function addApiKey(
   apiKey: ApiKey,
   openModal: (key: string) => void,
-  includeInvalid: boolean
+  includeExpired: boolean
 ): ThunkResult<void> {
   return async dispatch => {
     const result = await getBackendSrv().post('/api/auth/keys', apiKey);
     dispatch(setSearchQuery(''));
-    dispatch(loadApiKeys(includeInvalid));
+    dispatch(loadApiKeys(includeExpired));
     openModal(result.key);
   };
 }
 
-export function loadApiKeys(includeInvalid: boolean): ThunkResult<void> {
+export function loadApiKeys(includeExpired: boolean): ThunkResult<void> {
   return async dispatch => {
-    const response = await getBackendSrv().get('/api/auth/keys?include_invalid=' + includeInvalid);
+    const response = await getBackendSrv().get('/api/auth/keys?IncludeExpired=' + includeExpired);
```

Comment:
I personally think a camelCase query parameter would be more idiomatic than a PascalCase one. I don't know what our general style is though in the REST API. Writing PascalCase query params when making REST calls seems painful.

Reply:
I think the general style is camelCase; I just copied from the above comment.
I will change it.

Reply:
@papagian Thanks :)